gio/Gir.toml

```diff
@@ -284,6 +284,24 @@ generate_builder = true
     name = "run"
     manual = true
     doc_trait_name = "ApplicationExtManual"
+    [[object.function]]
+    name = "hold"
+    # Returns an value that releases on Drop
+    manual = true
+    doc_trait_name = "ApplicationExtManual"
+    [[object.function]]
+    name = "release"
+    manual = true
```

Comment:
What's the reason for this one?

Reply:
See 4 lines above

Reply:
And how someone without this PR context can guess it? ;)

Reply:
What does that have to do with the PR? The explanation is 4 lines above in the .toml file

Reply:
Because they don't seem linked to each other just from taking a look at it.